src/proto/active_packetizer.rs

```diff
@@ -383,29 +382,25 @@ where
                         if err.is_none()
                             || (opcode == request::OpCode::Exists && err == Some(ZkError::NoNode))
                         {
-                            trace!(logger, "pending watcher turned into real watcher"; "xid" => xid);
+                            trace!(xid, "pending watcher turned into real watcher");
                             this.watchers.entry(w.0).or_default().push((w.1, w.2));
                         } else {
-                            trace!(logger,
-                                   "pending watcher not turned into real watcher: {:?}",
-                                   err;
-                                   "xid" => xid
+                            trace!(
+                                xid,
+                                error = ?err,
+                                "pending watcher not turned into real watcher"
                             );
                         }
                     }
 
                     if let Some(e) = err {
-                        info!(logger,
-                               "handling server error response: {:?}", e;
-                               "xid" => xid, "opcode" => ?opcode);
+                        info!(xid, ?opcode, error = ?e, "handling server error response");
 
                         let _ = tx.send(Err(e));
                     } else {
                         let mut r = Response::parse(opcode, &mut buf)?;
 
-                        debug!(logger,
-                               "handling server response: {:?}", r;
-                               "xid" => xid, "opcode" => ?opcode);
+                        debug!(xid, ?opcode, "handling server response");
```

Comment:
question: Any particular reason we dropped `response` from this trace?

Reply:
I just missed it -> 421ac3d2e504272a26330e9b94cb924d915cf57b